MAX_ACCELERATION: 1.0
MAX_CURVATURE: 4.0
MAX_D_CURVATURE: 16.0
MAX_YAWRATE: 0.8
